CANscope/CZUL/CZUL.sct2

```diff
@@ -24343,6 +24343,10 @@ czul-czyz_SE_(X)CN N045.29.23.251 W076.14.59.928 N045.50.15.000 W076.16.00.000
 ;RW-HV
 RW-HV N047.35.18.777 W070.51.28.548 N046.52.21.731 W069.49.32.658
 ;ZUL-ZYZ Gates
+CZUL North Gate           N040.00.00.000 W080.00.00.000 N040.00.00.000 W080.00.00.000
```

Comment:
It's not my job to check for 'good' data, but it seems like this line just does nothing because both points are at the same coord?